templates/zerver/help/unsubscribe-stream.md

```diff
@@ -0,0 +1,40 @@
+# Unsubscribe from a stream
+
+If you want to stop receiving messages from a particular stream in Zulip, you can choose to unsubscribe from that stream. After you unsubscribe from a stream, you will no longer be able to receive, read, or send messages to the stream.
+
```

Comment:
The second sentence is slightly inaccurate, since you can still read old messages from an unsubscribed stream.  For simplicity let's just omit the sentence altogether, since the main purpose of this doc is how-to and not really about the semantics of unsubscribing.